services/invitations/requirements/_base.txt

```diff
@@ -25,6 +25,10 @@ anyio==3.6.2
     #   watchfiles
 async-timeout==4.0.2
     # via redis
+attrs==21.4.0
+    # via
+    #   -c requirements/../../../packages/service-library/requirements/././constraints.txt
```

Comment:
this is happening because service-library constraints were updated but the chance was not propagated downstream. Some other good example to discuss